scripts/zones/La_Theine_Plateau/npcs/Galaihaurat.lua

```diff
@@ -26,7 +26,7 @@ function onTrigger(player,npc)
 	if(player:getCurrentMission(SANDORIA) == THE_RESCUE_DRILL) then
 		local MissionStatus = player:getVar("MissionStatus");
 		
-		if(MissionStatus == 0) then
+		if(MissionStatus == 1) then

```

Comment:
So as it turns out, this broke it for me. When players talk to this guy, their var is in fact non existent, which should evaluate to zero. If your players mission status was somehow set to 1, something else was already wrong because nothing is setting this to one prior to getting to him.